internal/storage/v2/grpc/trace_storage.proto

```diff
@@ -62,6 +63,51 @@ message GetOperationsResponse {
   repeated Operation operations = 1;
 }
 
+message TraceQueryParams {
+  // service_name is a filter for traces that have been generated by a specific service.
+  string service_name = 1;
+  // operation_name is a filter for traces that have been generated by a specific operation.
+  string operation_name = 2;
+  // attributes contains a map of key-value pairs where the key is the name of the attribute
+  // and the value is the string-representation of the attribute value.
+  // Attributes are matched against the attributes of the resources and spans.
+  // At least one span in a trace must match all specified attributes.
+  map<string, string> attributes = 3;
+  // start_time is the start of the time interval for the query. All the returned traces
+  // will contain spans that have started after this time.
+  google.protobuf.Timestamp start_time = 4 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+  // end_time is the end of the time interval for the query. All the returned traces
+  // will contain spans that have started before this time.
+  google.protobuf.Timestamp end_time = 5 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
```

Comment:
@yurishkuro in api_v3 - these are referred to as `start_time_min` and `start_time_max`. I thought that was a little confusing and changed it to `start_time` and `end_time`. What do you think?

Reply:
+1

Reply:
the reason they are called so is to emphasize that it's a filter on the `start_time` field of a span.

Reply:
@yurishkuro Got it. Should I change this back then?

Reply:
I prefer to avoid divergence unless it's necessary. Here it's not necessary, and consistent naming has more benefits.